clang/lib/Driver/Driver.cpp

```diff
@@ -4631,9 +4632,12 @@ class OffloadingActionBuilder final {
       ActionList HostActionList;
       Action *A(HostAction);
       // Only check for FPGA device information when using fpga SubArch.
-      if (Args.hasArg(options::OPT_fintelfpga) &&
-          !(HostAction->getType() == types::TY_Object &&
-            isObjectFile(InputName))) {
+      auto SYCLTCRange = C.getOffloadToolChains<Action::OFK_SYCL>();
+      for (auto TI = SYCLTCRange.first, TE = SYCLTCRange.second; TI != TE; ++TI)
```

Comment:
```suggestion
      for (auto &TI = SYCLTCRange.first, TE = SYCLTCRange.second; TI != TE; ++TI)
```
(it might get parsed/deduced as per https://godbolt.org/z/ejvhb6)

Reply:
Doesn't look to be as easy as that.  The range from a const function so it will not allow me to increment it.  Uses of the offload change range is used in multiple places like this as it is.

Reply:
Thanks, got it! Let's disregard this idea then.